src/PIL/ImageFont.py

```diff
@@ -634,6 +634,7 @@ def getmask2(
         should have mode ``RGBA``. Otherwise, it should have mode ``1``.
 
         :param text: Text to render.
+        :param fill: A fill function.
```

Comment:
```suggestion
```
This has now been added in #6220 instead.

I wondered why it didn't conflict, until I realised it was because this is before `mode`, when it should be after.